Modix.Services/Utilities/FormatUtilities.cs

```diff
@@ -163,5 +163,76 @@ public static Color GetDominantColor(Image image)
 
             return (Color)mostCommonPaletteColor;
         }
+        
+        /// <summary>
+        /// Collapses plural forms into a "singular(s)"-type format.
+        /// </summary>
+        /// <param name="sentences">The collection of sentences for which to collapse plurals.</param>
+        /// <returns>A collection of formatted sentences.</returns>
+        public static IReadOnlyCollection<string> CollapsePlurals(IReadOnlyCollection<string> sentences)
```

Comment:
![bild](https://user-images.githubusercontent.com/22471295/54085672-c98deb80-4340-11e9-883a-7c11d732fafe.png)

You really weren't kidding huh 🤣

Reply:
Precisely